lmfit/minimizer.py

```diff
@@ -1353,7 +1353,7 @@ def emcee(self, params=None, steps=1000, nwalkers=100, burn=0, thin=1,
             _neg2_log_likel = -2*result.residual
 
             # assumes that residual is properly weighted
-            result.chisqr = np.exp(_neg2_log_likel)
+            result.chisqr = np.exp(min(650, _neg2_log_likel))
```

Comment:
This should get a comment

Reply:
@Tillsten just avoiding potential overflows.  I'll add a comment to the code